cogs/commands.py

```diff
@@ -376,6 +376,23 @@ async def suggestion(self, ctx, *, suggestion: str):
         msg = await ctx.send(embed=em)
         await msg.add_reaction('👍')
         await msg.add_reaction('👎')
+                       
+    @commands.Cog.listener()
+    async def on_raw_reaction_add(self, payload):
+        user = await self.client.fetch_user(payload.user_id)
```

Comment:
No need to fetch the user each time. `reaction_add` includes `payload.member` (If in a guild) which you can use to check if they are a bot.